.github/workflows/test-action.yml

```diff
@@ -0,0 +1,19 @@
+name: Test main.py
+on:
+  push:
+  pull_request:
+    branches: [ main ]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    services:
+      docker:
+        image: docker:19.03.12
+        options: --privileged
+    steps:
+    - uses: actions/checkout@v2
```

Comment:
Is there a reason why we should be using v2? It uses node 12 which is deprecated, and there are v3 and v4.

Reply:
Hah, the dangers of having an LLM generate code for you 😅